examples/chat/tui/actions.nim

```diff
@@ -309,7 +309,7 @@ proc action*(self: ChatTUI, event: UserMessageEvent) {.async, gcsafe, nimcall.}
 
     # if event.topic is not a properly formatted waku v2 content topic then the
     # whole string will be passed to printMessage
-    if topicSplit.len == 5:
+    if topicSplit.len == 5 and topicSplit[0] == "":
       # for "/toy-chat/2/example/proto", topic would be "example"
       topic = topicSplit[3]
```

Comment:
Even here, I think we could parse a `ContentTopic` object, or perhaps `UserMessageEvent.topic` could be of type `ContentTopic` and it would be parsed in the task.